src/avt/QtVisWindow/vtk/vtkRubberBandMapper2D.C

```diff
@@ -394,5 +401,173 @@ vtkRubberBandMapper2D::RenderOverlay_Qt(vtkViewport* viewport, vtkActor2D* actor
     double* actorColor = actor->GetProperty()->GetColor();
     SET_FOREGROUND_D(actorColor);
 
-#include <vtkRubberBandMapper2D_body.C>
+    int numPts;
```

Comment:
Note: the code starting at this line was copied over from vtkRubberBandMapper2D_body.C. The only change is the addition and use of "devicePixelRatio".